docs/HowToSupportProtectedBuild.md

```diff
@@ -113,15 +139,54 @@ The following sections give detailed explanation regarding some of the steps / c
 	*(.usrheapptr)		/* User heap pointer added in the .text user section */
 	....
 	_etext = ABSOLUTE(.);
-	} > uflash
+} > uflash
+```
+
+#### Make.defs
+- The make.defs file provides several definitions which are used during the build process. One of the definitions is (ARCHSCRIPT) for the linker script to be used to link the kernel binary.
+- Make.defs file needs to be modified to use these two linker scripts for protected build support.
```

Comment:
The file location is missing. Could you add where this should be added?

Reply:
Above files are the same. Could you add the location where to be added?

Reply:
added